src/rules/top-level-interop.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @fileoverview Rule to ensure top-level items are either an array or ojbect.
+ * @author Joe Hildebrand
+ */
+
+export default {
+	meta: {
+		type: /** @type {const} */ ("problem"),
+
+		docs: {
+			description:
+				"Disallow JSON top-level items are not an array or object",
+		},
+
+		messages: {
+			topLevel:
+				'Top level item should be array or object, got "{{type}}".',
```

Comment:
We tend to use single quotes to surround values in message.

```suggestion
				"Top level item should be array or object, got '{{type}}'.",
```